src/main/java/uk/co/codezen/maven/redlinerpm/mojo/AbstractRpmMojo.java

```diff
@@ -108,6 +108,12 @@
     protected boolean performCheckingForExtraFiles = true;
 
     /**
+     * Whether to replace already existing artifacts
+     */
+    @Parameter
+    protected boolean shouldReplaceArtifact = false;
```

Comment:
I was just trying to be unobtrusive, so didn't want to change the default behavior of the plugin.  But I agree.  I'll wait to hear from @spaulg before updating.